docs/cudf/source/cudf_pandas/how-it-works.md

```diff
@@ -36,3 +36,16 @@ transfers.
 When using `cudf.pandas`, cuDF's [pandas compatibility
 mode](api.options) is automatically enabled, ensuring consistency with
 pandas-specific semantics like default sort ordering.
+
+
+`cudf.pandas` uses a managed pool memory by default, that will also enable
+prefetching <<LINK to prefetch tutorial/demo>>.
```

Comment:
@vyasr Would you be able to add a tutorial link/demo about prefetch that might be appropriate for cudf users?

Reply:
I'm not sure. What kind of information would you want to convey? How prefetching works from the perspective of CUDA? How a Python users would trigger it (in this case, they wouldn't do so themselves)?

Reply:
I was thinking more like this: https://resources.nvidia.com/en-us-nsight-developer-tools/boosting-application?lx=P1ZhhI&topic=hpc&linkId=100000269652539&ncid=no-ncid

Reply:
I think explaining prefetching requires first explaining what CUDA Unified Memory is, what virtual memory paging is, what a page fault is and why we want to avoid them, and then how prefetching helps avoid them.  That's a lot. I think we should just link to the best post or documentation we have that covers all of that. 

This one seems to fit the bill: https://developer.nvidia.com/blog/improving-gpu-memory-oversubscription-performance/

Reply:
Thanks @harrism ! That is a perfect !!